Resources/Base.lproj/TSSTSessionWindow.xib

```diff
@@ -111,6 +111,27 @@
                             </connections>
                         </segmentedControl>
                     </toolbarItem>
+                    <toolbarItem implicitItemIdentifier="D01E117F-B172-435D-8B8C-CFC587DA6020" label="Page End" paletteLabel="Page End" id="dMP-Th-vKR" customClass="DTPageTurnToolbarItem">
+                        <nil key="toolTip"/>
+                        <size key="minSize" width="51" height="25"/>
+                        <size key="maxSize" width="59" height="27"/>
+                        <segmentedControl key="view" verticalHuggingPriority="750" id="U2R-Pp-TY8">
+                            <rect key="frame" x="0.0" y="14" width="59" height="27"/>
+                            <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMinY="YES"/>
+                            <segmentedCell key="cell" borderStyle="border" alignment="left" style="separated" trackingMode="momentary" id="msg-5l-0tY">
+                                <font key="font" metaFont="system"/>
+                                <segments>
+                                    <segment toolTip="Left Page End" image="NSTouchBarSkipToStartTemplate" width="24" tag="701">
```

Comment:
Probably needs a better tooltip than "Left Page End". Maybe "Left End Page"?

Reply:
Set to "Left Page End"